explorePy/variable_summary.py

```diff
@@ -2,15 +2,57 @@
 import pandas as pd
 
 
-
-def variable_summary(x):
+def variable_summary(frames):
     """
     takes a data frame as input and provides the total quantity of each type of variable present in the data frame
-
     inputs:
-    x: data frame
-
+    frames: data frame
     returns:
     data frame - variable type and count of each type
     """
-    return x.variable_summary()
+    #https://stackoverflow.com/questions/19900202/how-to-determine-whether-a-column-variable-is-numeric-or-not-in-pandas-numpy
+    #pandas data types
+    #https://pbpython.com/pandas_dtypes.html
```

Comment:
we can delete this line (15)

Reply:
That is true, a print statement will suffice.